src/ServiceControlInstaller.Engine/Instances/ServiceControlInstance.cs

```diff
@@ -428,6 +428,23 @@ public void UpdateDatabase(Action<string> updateProgress)
             }
         }
 
+        public void RemoveDatabaseIndexes()
+        {
+            var folders = AppConfig.RavenDataPaths().ToList();
+
+            foreach (var folder in folders.OrderByDescending(p => p.Length))
+            {
+                try
+                {
+                    FileUtils.DeleteDirectory(Path.Combine(folder, "Indexes"), true, false);
```

Comment:
I think you also need to delete IndexDefinitions folder. I can very tomorrow

Reply:
Upgrading the small database confirmed that we don't have to delete the IndexDefinitions folder. Going to merge